types/foundry-vtt-api/types/core/die.d.ts

```diff
@@ -0,0 +1,197 @@
+/**
+ * The base Die class.
+ *
+ * Each Die instance represents a distinct term in a roll equation which transacts rolls of an die with some number
+ * of faces. The Die instance provides controls for rerolling, exploding, counting, or modifying the set of results
+ * from the Die.
+ *
+ * @param {Number} faces    The number of faces for this Die
+ *
+ * @example
+ * // Define a 6-sided die
+ * let die = new Die(6);
+ *
+ * // Roll the die 4 times
+ * die.roll(4);
+ *
+ * // Roll another 2 times, adding the new results to the existing set
+ * die.roll(2);
+ *
+ * // For all 6 of the initial rolls, reroll if any result was a 1
+ * die.reroll([1]);
+ *
+ * // For set of remaining results, roll a bonus die if any result was a 6
+ * die.explode([6]);
+ *
+ * // Count the total number of rolls which was greater than 3
+ * die.countSuccess(3, ">");
+ *
+ * // Display the total number of successes
+ * console.log(die.total);
+ */
+declare class Die {
+	/**
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/core/die.d.ts;
  Line: 33;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.